### PR TITLE
Export content from MSDN

### PR DESCRIPTION
**Pull request by yishengjin1413**
test MSDN page to ECMAXxml migration

### CODE REVIEW

**Comment by yishengjin1413** [comment]
Added exceptions to ecmaxml in new commit.

**Comment by mairaw** [comment]
@yishengjin1413 one thing I noticed for example is that we're losing some tag values in this process.

For example, it's exporting as **null** instead of <see langword="null" /> or *s* instead of <paramref name="s" />. Is this because we're scraping from MSDN so you no longer have this formatting info?

**Comment by yishengjin1413** [comment]
@mairaw Do you mean the **null** in this [page](https://msdn.microsoft.com/en-us/library/IEHost.Execute.IEExecuteRemote.Exception(v=VS.90))? Yes. After export the MD files from MSDN page, we just get a plain text instead of a bold tag. Do you think it's a block issue we need to fix?

File: xml/System.Web.Handlers/ScriptModule.xml [hunk 1]
```diff
@@ -104,17 +104,21 @@
         <Parameter Name="app" Type="System.Web.HttpApplication" Index="0" FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8" />
       </Parameters>
       <Docs>
-        <param name="app">An <see cref="T:System.Web.HttpApplication" /> object that provides access to the methods, properties, and events that are common to all applications in an ASP.NET application.</param>
+        <param name="app">An [HttpApplication](https://msdn.microsoft.com/en-us/library/4wt3wttw\(v=vs.110\)) object that provides access to the methods, properties, and events that are common to all applications in an ASP.NET application.</param>
```
Comment (ChrisMaddock) [inline xml/System.Web.Handlers/ScriptModule.xml]: Just being nosy - this looks to be changing a `<see>` link to Markdown - something which we've just been correcting in the opposite direction in https://github.com/dotnet/docs/issues/1931. Is this expected, as something to get the basic data in, and then fix later?

Comment (mairaw) [reply xml/System.Web.Handlers/ScriptModule.xml]: I appreciate the feedback @ChrisMaddock! This PR shouldn't be modifying what we already have. Let me take a closer look.

File: xml/System.ComponentModel.Design.Serialization/DesignerSerializationManager.xml [hunk 2]
```diff
@@ -1045,8 +1045,18 @@
         <ReturnType>System.ComponentModel.Design.Serialization.ResolveNameEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Occurs when [IDesignerSerializationManager.GetName](https://msdn.microsoft.com/library/bb343362) cannot locate the specified name in the serialization manager's name table.</summary>
```
[the remark this answers is not in the document]
Comment (mairaw) [reply xml/System.ComponentModel.Design.Serialization/DesignerSerializationManager.xml]: Yes, I've noticed that @ChrisMaddock. 😢 Does your script can convert from these Markdown links with see cref links? I've been trying to fix these as I review this PR. For example:
image

[the remark this answers is not in the document]
Comment (ChrisMaddock) [reply xml/System.ComponentModel.Design.Serialization/DesignerSerializationManager.xml]: Oh, no sorry. It just converts to `href` links 😢

[the remark this answers is not in the document]
Comment (mairaw) [reply xml/System.ComponentModel.Design.Serialization/DesignerSerializationManager.xml]: No worries @ChrisMaddock! We can identify the Markdown links post PR then and I'll try to fix the cref links if not too many.

**Comment by mairaw** [comment]
Thanks @yishengjin1413. I've replace this PR with smaller PRs to keep my sanity. 😄

**Pull request closed without merging**